apps/studio/components/interfaces/Storage/CreateBucketModal.tsx

```diff
@@ -95,12 +95,26 @@ const formId = 'create-storage-bucket-form'
 
 export type CreateBucketForm = z.infer<typeof FormSchema>
 
-export const CreateBucketModal = () => {
+interface CreateBucketModalProps {
+  hideAnalyticsOption?: boolean
```

Comment:
Backwards compatibility. Since this dialog in Storage V2 is just for file buckets, we don’t need the below `"type"` part of the form.

Reply:
Storage V2 will have a different dialog for analytics (and vector) buckets.

Reply:
i reckon we actually don't need a prop here - could just listen to the feature preview

since only the new UI will need this to be true